src/index.js

```diff
@@ -0,0 +1,33 @@
+import './style.css';
+
+const todosListEl = document.getElementById('todos-list');
+
+const todos = [
+  {
+    description: 'task1',
+    isCompleted: true,
+  },
+  {
+    description: 'task2',
+    isCompleted: false,
+  },
+  {
+    description: 'task3',
+    isCompleted: false,
+  },
+];
+
+let content = '';
+// RENDER TODOS
+todos.forEach((todo, index) => {
+  content += `
+    <li id="todo-item">
+      <div id=${index}>
+        <span class="bi-circle"></span>
+        <p>${todo.description}</p>
+        <p>${todo.isCompleted}</p>
+      </div>
+    </li>`;
+});
```

Comment:
- [ ] These should be check boxes not list item identifiers. Kindly fix that.

  ![image](https://github.com/Zohra-Neda/To-Do-List/assets/81468675/a867664d-eea6-4861-8992-33732990d7ea)